pkg/controller/config/match/match.go

```diff
@@ -0,0 +1,81 @@
+package match
```

Comment:
we should name this package after the excluder object.

Reply:
do you mean rename this package `processExcluder`?

Reply:
or `processexcluder`? I'm not in love with the no-casing approach, but it seems to be what other golang packages are doing.

Reply:
the biggest thing is `match.GetSet()` doesn't really say what the code is doing. `excluder.Get()` (or similar), conveys a bit more meaning.

Reply:
updated